rails-snow-queen/db/migrate/20181201233429_create_quotes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateQuotes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :quotes do |t|
+      t.string :address
+      t.float :area
+      t.decimal :total
```

Comment:
doesn't this need a `precision` as well?

Reply:
Will address this after the presentation